etc/systemd/system/multi-user.target.wants/cernvm-contextualization.service

```diff
@@ -0,0 +1 @@
+/usr/lib/systemd/system/cernvm-contextualization.service
```

Comment:
I think this file is not necessary but we need to add the service to `enable_services()` in etc/cernvm/postinstall